dotCMS/src/main/java/com/dotmarketing/portlets/workflows/actionlet/DestroyContentActionlet.java

```diff
@@ -0,0 +1,58 @@
+package com.dotmarketing.portlets.workflows.actionlet;
+
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionClassParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionFailureException;
+import com.dotmarketing.portlets.workflows.model.WorkflowActionletParameter;
+import com.dotmarketing.portlets.workflows.model.WorkflowProcessor;
+import com.dotmarketing.util.Logger;
+import com.liferay.portal.language.LanguageUtil;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Actionlet to destroy a contentlet
+ * @author jsanca
+ */
+@Actionlet(destroy = true)
+public class DestroyContentActionlet extends WorkFlowActionlet {
+    
+    private static final long serialVersionUID = -2314685590620626801L;
+
+    @Override
+    public List<WorkflowActionletParameter> getParameters() {
+        return null;
+    }
+    
+    @Override
+    public String getName() {
+        return "Destroy content";
+    }
+    
+    @Override
+    public String getHowTo() {
+        return "This action will destroy the content. Warning: this can't be undone!";
```

Comment:
this will destroy the content in ALL languages that exist and cannot be undone

Reply:
Done